CryptoPrices/Sources/Domain/Tests/UseCasesTests/MyCoinsUseCase/MyCoinsUseCaseTests.swift

```diff
@@ -1,11 +1,54 @@
+import DomainTestHelpers
 import Nimble
 import Quick
+import TestHelpers
 @testable import UseCases
 
 final class MyCoinsUseCaseTests: QuickSpec {
```

Comment:
This is optional. I think using the suffix `Spec` is suitable in this case. Because when using lib Nimble & Quick, we used the suffix `Spec` for consistency as [a convention](https://nimblehq.co/compass/development/code-conventions/swift/quick-nimble/#convention-1). I think it makes sense for the suffix `Tests` when we conform XCTestCase :D.

I also think it can be flexible and understood by developers with the suffix "Tests". 😄